src/Infrastructure.Dapper/Repositories/UserRepository.cs

```diff
@@ -114,9 +127,15 @@ public async Task<DateTime> GetAccountRevisionDateAsync(Guid id)
             return results.SingleOrDefault();
         }
     }
+    public override async Task<User> CreateAsync(User user)
```

Comment:
❓ Is it standard to have a newline above?

Reply:
yes, will fix